scripts/cert_generation.sh

```diff
@@ -265,3 +299,13 @@ generate_certs 'system.admin'
 generate_certs 'kibana-internal' "$(generate_extensions false false kibana)"
 generate_certs 'elasticsearch' "$(generate_extensions true true $LOG_STORE{,-cluster}{,.${NAMESPACE}.svc}{,.cluster.local})"
 generate_certs 'logging-es' "$(generate_extensions false true $LOG_STORE{,.${NAMESPACE}.svc}{,.cluster.local})"
+
+if [ ! -f "${WORKING_DIR}/kibana-session-secret" ] || [ "$(du -b "${WORKING_DIR}/kibana-session-secret" | cut -f1)" = 0 ] ; then
+  info "Generating kibana session secret"
+  python -c "import string,random;print(''.join(random.choices(string.ascii_letters + string.digits, k=32)))" >  "${WORKING_DIR}/kibana-session-secret"
+fi
+
+if [ -f ${WORKING_DIR}/action.log ] ; then
+  cat ${WORKING_DIR}/action.log
+  rm ${WORKING_DIR}/action.log
+fi
```

Comment:
Missing newline here

Reply:
Added... but why does it matter if there is a newline after `fi`

Reply:
It makes GitHub's diff happy :)